app/__init__.py

```diff
@@ -94,6 +94,10 @@ def create_app(application, config=None):
     register_blueprint(application)
     register_v2_blueprints(application)
 
+    # Log the application configuration
+    config_values = config.get_config(config.get_sensitive_config())
```

Comment:
Minor improvements: I'd just have a one function call to simplify on the API consumer side, which can include this this exact line wrapped in a function such as `config.get_safe_config())`.

Reply:
Ah yeah - I like that idea.  Added!